test/colony-network-mining.js

```diff
@@ -1506,8 +1544,12 @@ contract("ColonyNetworkMining", accounts => {
       await badClient.initialise(colonyNetwork.address);
       await goodClient.addLogContentsToReputationTree();
       await badClient.addLogContentsToReputationTree();
+
+      await forwardTime(3540, this);
       await goodClient.submitRootHash();
       await badClient.submitRootHash();
+      await forwardTime(60, this);
```

Comment:
Any reason why you've squeezed in the hash submissions into smaller time here than the other tests?

Reply:
No reason. I think this is just a relic from initial tinkering. Will change to 3500 which is at least consistent with a lot of the others. Even doing that though, looks like we've ended up with a split between skips of 3500/100 and 1800/1800... shall I make those consistent where possible (which should be most places)?

Reply:
yes please! Considering we'll be likely changing this timeframe to more than an hour, it maybe worth extracting this logic of submissions into a helper like here https://github.com/JoinColony/colonyNetwork/pull/357/files#diff-15aaaaa81a55d904a938772d3e2a3a82R2855?